lib/db_client.py

```diff
@@ -0,0 +1,92 @@
+import os
+import psycopg2
+import redshift_connector
+
+from botocore.exceptions import ClientError
+from enum import Enum
+from helpers.log_helper import create_log
+
+
+class DbMode(Enum):
+    SIERRA = 1
+    REDSHIFT = 2
+
+    def __str__(self):
+        return self.name.lower()
+
+
+class DbClient:
```

Comment:
Just an architectural suggestion: It might be nice to create a base `Client` class here and subclass it for psql and redshift. Just in anticipation of wanting/needing to make the specific implementation of these things more complex.

Reply:
If/when the two clients begin to differ, I'll definitely do this, but for now I think keeping them identical is sufficient.